src/hotspot/os/aix/os_aix.cpp

```diff
@@ -2222,6 +2223,15 @@ static void set_page_size(size_t page_size) {
 
 // This is called _before_ the most of global arguments have been parsed.
 void os::init(void) {
+  if (UseSystemMemoryBarrier) {
+    if (!SystemMemoryBarrier::initialize()) {
+      if (!FLAG_IS_DEFAULT(UseSystemMemoryBarrier)) {
```

Comment:
Have we processed this flag by this stage?

Reply:
This is the leftover. See below.